hooks/pre-push

```diff
@@ -50,7 +50,7 @@ bash scripts/run_linter.sh
 check_test_status "Linter"
 
 echo "Running karma tests"
-bash scripts/run_karma_tests.sh --browsers PhantomJS
```

Comment:
I had got around that by using es6-shim ([here](https://www.npmjs.com/package/es6-shim)). [This line](https://github.com/google/tie/blob/master/karma.conf.js#L33) should ensure that errors are not brought up for the incompatibility for any of the questions. If there are other directories that use es6 we can include them there. The 2 lines above that may need to be uncommented for this (comment was added [here](https://github.com/google/tie/commit/1fa187b21eddbdc72f6060e21f2bf81fb02f8f3f#diff-a2a3b7b0c9c3b4b93b4aebf4e3ec3cfb)).